compiler/z/codegen/OMRRegisterDependency.cpp

```diff
@@ -481,6 +501,119 @@ TR::RegisterDependencyConditions  *OMR::Z::RegisterDependencyConditions::clone(T
    return other;
    }
 
+
+uint32_t
+OMR::Z::RegisterDependencyConditions::setNumPreConditions(uint16_t n, TR_Memory * m)
+   {
+   if (_preConditions == NULL)
+      {
+      _preConditions = new (n,m) TR::RegisterDependencyGroup;
+   for(int32_t i=0;i<n;i++)
+      {
+      _preConditions->clearDependencyInfo(i);
+      }
```

Comment:
Formatting.

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/9858655e8793d27c54d7d0dad9e11c2f2ffe3f65..896d27b318c12e5c7b8eacb92b4a8f297f0842cc).